src/main/resources/templates/db_scripts/update_user_tenants_table.sql

```diff
@@ -4,7 +4,8 @@ ALTER TABLE user_tenant
   ADD COLUMN IF NOT EXISTS mobile_phone_number text,
   ADD COLUMN IF NOT EXISTS phone_number text,
   ADD COLUMN IF NOT EXISTS barcode text,
-  ADD COLUMN IF NOT EXISTS external_system_id text;
+  ADD COLUMN IF NOT EXISTS external_system_id text,
+  ADD COLUMN IF NOT EXISTS consortium_id text;
```

Comment:
uuid type should be here instead of text

Reply:
Changed to uuid